small nit: fix unmeaningful log

WeiZhang555 opened the pull request:
Truncated dir name can't give any useful information, print whole dir
name will.

Bad debug log is like this:

```
DEBU[2449] aufs error unmounting /var/lib/doc: no such file or directory 
DEBU[2449] aufs error unmounting /var/lib/doc: no such file or directory 
DEBU[2449] aufs error unmounting /var/lib/doc: no such file or directory 
DEBU[2449] aufs error unmounting /var/lib/doc: no such file or directory 
DEBU[2449] aufs error unmounting /var/lib/doc: no such file or directory
```

it fulfills log file but provides nothing meaningful.

ping @cpuguy83 , who last modified this line.

Signed-off-by: Zhang Wei zhangwei555@huawei.com

thaJeztah commented:
LGTM

thaJeztah commented:
also wondering if we should ignore the error if the path doesn't exist; @cpuguy83 ?

cpuguy83 commented:
LGTM

I don't want to ignore. It helps signal a problem, and since its debug only it's fine.

thaJeztah commented:
oh! you're right, it's debug only, sgtm

thaJeztah commented:
adding this to the milestone as it can help us resolving possible issues

WeiZhang555 committed:
small nit: fix unmeaningful log

Truncated dir name can't give any useful information, print whole dir
name will.

Bad debug log is like this:

```
DEBU[2449] aufs error unmounting /var/lib/doc: no such file or directory
```

Signed-off-by: Zhang Wei <zhangwei555@huawei.com>
